.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "\U00002753 Crystal Community Forum"
+    url: https://forum.crystal-lang.org/c/help-support
+    about: Please ask and answer questions here.
```

I'd phrase this more concrete to show what kind of questions should go there. 
```suggestion
    about: Questions about Crystal. You'll find help for installing Crystal, using it and any issues you might encounter.
```

---

Seems a bit too long.

---

I pushed an alternative sentence, inspired by yours.